### DISCUSSION TITLE
GHC 9.10, base-4.20 not accepted

### DISCUSSION THREAD

**Discussion by ysangkok** [chrisdone/lucid#155] [General]
This version of GHC is excluded by the bounds, which I believe could be bumped with a revision.

**Comment by chrisdone** [comment]
Sounds good, I’ll bump it this morning.

**Comment by chrisdone** [comment]
Looks like GHC 9.10 is not on stackage yet, not even nightly, so I can't easily test it. Once it's on (GHC 9.10), I will test it and then push a version bump.

**Comment by ysangkok** [comment]
Ok, I can understand why you'd prefer Stack, but it's surprising since [the last bump is from October 2023](https://github.com/chrisdone/lucid/issues/145), which is two months before [Stackage Nightly became available with GHC 9.8](https://www.stackage.org/blog/2024/01/announce-lts-22-nightly-ghc-9.8). Is there something that makes this unlike the situation with #145?

**Comment by chrisdone** [comment]
In this situation I feel like testing it myself.

**Comment by ysangkok** [comment]
Just as a note to anyone watching this: you can use lucid-2.9.9 in your extra-deps, since it has bounds `base >= 4.5 && <5` which is compatible with GHC 9.10. This version is from 2017, but it seems to work reasonably well.

**Comment by chrisdone** [comment]
This one is applied (base<4.21) 
https://hackage.haskell.org/package/lucid-2.11.20230408/revisions
https://hackage.haskell.org/package/lucid2-0.0.20240424/revisions
